class.md

```diff
@@ -24,6 +24,10 @@
 
   -[Constance](https://github.com/coco390) Frontend Development
 
+- ### **D**
+
+  -[Dominique STEVE](https://github.com/dsteve) Meta Frontend Development Professional Certification
```

Comment:
The correct training name is Meta Frontend Developer Professional Certification
Change Development to Developer, please